src/main/java/com/cocos/cocos/api/review/service/ReviewService.java

```
@@ -249,8 +257,20 @@ public HospitalReviewListResponse getHospitalReviewList(final Long hospitalId, f
                     final Breed breed = breedMap.get(review.getBreedId());
                     final Member member = memberMap.get(review.getMemberId());
                     final Pet pet = petMap.get(review.getMemberId());
+
+                    if (hospital == null || breed == null || member == null || pet == null) {
```

p2; 해당 부분이 필요한 이유가 궁금합니다.

---

정상적인 응답으로 변환할 수 없는 리뷰는 건너뛰기 위해 map 단계에서 null을 반환했습니다!

---

고민해봤는데 에러가 난 경우 빠르게 파악하는 게 더 중요할 것 같아, 에러를 던지는 방식으로 변경했습니다 (fail fast)